gulpfile.js

```diff
@@ -8,6 +8,12 @@ var testPath = "./test";
 var binPath = "./bin";
 var tsFiles = "/**/*.ts";
 
+var androidEmulatorName = "emulator";
+var iOSEmulatorName = "iPhone 6s Plus (9.3) [";
```

Comment:
Again, we should not hardcode these values, but pick one of the already installed emulator/simulator on the machine.